src/util/irep_ids.def

```diff
@@ -665,6 +665,7 @@ IREP_ID_TWO(java_lambda_method_handles, lambda_method_handles)
 IREP_ID_ONE(havoc_object)
 IREP_ID_TWO(overflow_shl, overflow-shl)
 IREP_ID_TWO(C_no_initialization_required, #no_initialization_required)
+IREP_ID_TWO(C_no_nondet_initialization_allowed, #no_nondet_initialization_allowed)
```

Comment:
A bit of nit-picking: "allowed" - who sets those rules? Do I get jailed if I do a nondet-init? :-)  More seriously: The commit message, which is the only documentation here, provides examples, but doesn't really set out the rules. Effectively the commit message will need to set out the semantics.

Reply:
> Do I get jailed if I do a nondet-init? 

You get punished with false alarms.

Reply:
I'll leave out the `allowed` to make it clear - just no nondet-init, period, don't even try :) I will also amend the commit message in lines with my last message - marking internal variables that are not CPROVER variables, language specific internal variables.